### ISSUE TITLE
Unable to report security issue

### ISSUE THREAD

**Issue by yohanes** [intel/ittapi#165]
I have tried reporting according to this: https://github.com/intel/ittapi/blob/master/SECURITY.md

But I got auto reply that my email is being forwarded to another email:       

Security issues with Intel managed open source projects:  [secure-opensource@01.org](mailto:secure-opensource@01.org)

I tried emailing directly to secure-opensource@01.org, it bounced, and no MX is set for this domain.

I never got any reply for this. I tried reporting via Twitter/X, https://x.com/yohanes/status/1849403767713927189, they just assured me that they will get back to me (never hapened). 

The issue affects the android version, and this library is used by default by OpenCV. 

Pleas advise, should I just post the bug here, and let everyone handle the problem themself?

**Comment by abrown** [comment]
@yohanes, I think you're in the right place now to get the attention of the developers of this project (cc: @ekovanova, @eparshut). And thanks for reporting whatever you found. I think the best way to report the vulnerability is through GitHub's "Report a vulnerability" page: https://github.com/intel/ittapi/security/advisories/new. Does that work for you?

**Comment by abrown** [comment]
And I wouldn't worry too much about the CVSS rating and all that, just report what you found.

**Comment by yohanes** [comment]
Thank you for the info, I have submitted the bug now through https://github.com/intel/ittapi/security/advisories/new

**Comment by abrown** [comment]
Thanks! There's an internal e-mail chain and someone should now take a look.

**Comment by abrown** [comment]
I'll close this since I think we at least have the initial report.

**Comment by llindsaya** [comment]
@yohanes you were correct in how you initially reported your findings based on https://github.com/intel/ittapi/blob/master/SECURITY.md.  Please continue to use the guidance there in the future.

**Comment by yohanes** [comment]
I did follow the guidance perfectly six weeks ago. I immediatelly got an auto reply, but never got any human reply. That's why I posted here. If that is the correct channel, I don't know how long I need to wait.

I really suspect something is wrong with your mailbox for opensource project reporting. The auto reply says: 

> PSIRT will forward your email without a response as follows (save this information for future reference):
> * Security issues with Intel managed open source projects:  [secure-opensource@01.org](mailto:secure-opensource@01.org)

Try emailing that address, it will always bounce Because an MX record was not set for that domain, every email will try to connect to the the primary address listed in A record, and none of the email ports are open for that IP, so no email will reach there.
